app/helpers/observer_helper.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
```

Comment:
This file was renamed from ShowObservationHelper in order to comply with Rails conventions. The new name makes testing much easier.